extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -515,6 +517,7 @@ export class AmpSlideScroll extends BaseSlides {
           setStyle(this.slideWrappers_[i], 'order', '');
         }
         this.slideWrappers_[i].classList.remove(SHOWN_CSS_CLASS);
+        this.slides_[i].setAttribute('aria-hidden', 'true');
```

Comment:
SGTM. End result is the same,`removeAttribute` is a faster DOM operation than `setAttribute` anyway.